pydatastructs/linear_data_structures/linked_lists.py

```diff
@@ -37,12 +37,12 @@ class DoublyLinkedList(LinkedList):
 
     >>> from pydatastructs import DoublyLinkedList
     >>> dll = DoublyLinkedList()
-    >>> dll.append(6)
+    >>> dll.append_right(6)
     >>> dll[0].data
     6
     >>> dll.head.data
     6
-    >>> dll.append(5)
```

Comment:
IMO, `append` is fine. Python, `list` also uses some similar API.

Reply:
OK. I did it for symmetry. Will fix it in the next commit for **CircularLinkedList**.

Reply:
Please add `CircularLinkedList` in a different PR. Otherwise it will be difficult to review.